### ISSUE TITLE
Is it possible to generate ILGenerator.Emit calls?

### ISSUE THREAD

**Issue by wmjordan** [segrived/Msiler#35]
This extension is great.
I am working with `DynamicType` and `DynamicMethod`s. Could you please add an _Emit_ view which shows corresponding `ILGenerator.Emit` calls for the selected method?

**Comment by segrived** [comment]
Do you mean something like this, right?

```
ilGen.Emit(OpCodes.Ldc_I4_3)
ilGen.Emit(OpCodes.Stloc_0)
```

instead of

```
IL_0001 ldc.i4.3
IL_0002 stloc.0
```

**Comment by wmjordan** [comment]
Yeah yeah yeah.

**Comment by segrived** [comment]
Yes, nice idea, i like it. At this time i can't implement this feature, maybe in future versions. Thanks for idea, i will keep this issue opened.

**Comment by wmjordan** [comment]
Thank you very much for the understanding and all efforts.
I am writing a JSON serializer and I am attempting to introduce more dynamic methods to make the serialization and deserialization process faster. We all knows that dynamic methods are error-prone and hard to create or debug. If you can implement such a function, creating dynamic methods will be a breeze.
